### PR TITLE
Renamed Ignition and ign to Gazebo or gz

### PR DESCRIPTION
**Pull request by ahcorde**
Signed-off-by: ahcorde <ahcorde@gmail.com>

# 🦟 Bug fix

## Summary

Related with 
 - https://github.com/gazebosim/garden-tutorial-party/issues/1917 
 - https://github.com/gazebosim/garden-tutorial-party/issues/1885
 - https://github.com/gazebosim/garden-tutorial-party/issues/1893
 - https://github.com/gazebosim/garden-tutorial-party/issues/1877

Renamed Ignition and ign to Gazebo or gz.

## Checklist
- [ ] Signed all commits for DCO
- [ ] Added tests
- [ ] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)
- [ ] Consider updating Python bindings (if the library has them)
- [ ] `codecheck` passed (See [contributing](https://gazebosim.org/docs/all/contributing#contributing-code))
- [ ] All tests passed (See [test coverage](https://gazebosim.org/docs/all/contributing#test-coverage))
- [ ] While waiting for a review on your PR, please help review [another open pull request](https://github.com/pulls?q=is%3Aopen+is%3Apr+user%3Agazebosim+archived%3Afalse+) to support the maintainers

**Note to maintainers**: Remember to use **Squash-Merge** and edit the commit message to match the pull request summary while retaining `Signed-off-by` messages.

### CODE REVIEW

File: install/tutorial_6-x.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ Install prerequisites.  A clean Ubuntu system will need:
 
     sudo apt-get install ruby-dev build-essential libtinyxml-dev libboost-all-dev cmake mercurial pkg-config
 
-### ignition-math4:
+### gz-math4:
```
Comment (azeey) [inline install/tutorial_6-x.md:25]: Do we have a `gz-math4`? I think it's better to leave it as `ignition` for older tutorials

File: quickstart/tutorial_2-0.md [hunk 2]
```diff
@@ -154,7 +154,7 @@ std::cout << "Joint " << jointName << " connects " << parentLinkName
...
+As you might guess, we're getting the `name` attribute of the current `joint_element` and then retrieving its `parent` and `child` elements' values. It is interesting to note all `Get()` method's use cases: `Get()` is a template method that returns the given attribute value or the element value itself (that is, the plain text in between tags) if called with no arguments. To this end, it has been specialized to parse such value accordingly into primitive types (i.e. `double`), common std types (i.e. `std::string`) or [`gazebo::math`](http://gazebosim.org/libraries/math) types for the most complex ones (like poses).
```
Comment (azeey) [inline quickstart/tutorial_2-0.md:157]: Since this is for an earlier version of `libsdformat`, having `ignition::math` might make more sense.

File: python_example.py [hunk 3]
```diff
@@ -0,0 +1,68 @@
+
+from gz.math7 import Vector3d
```
Comment (azeey) [inline python_example.py:2]: Was this file added intentionally? It doesn't seem relevant to the PR

File: install/tutorial_9_later.md [hunk 4]
```diff
@@ -180,22 +180,22 @@ Miniconda suffices.
...
+Install Gazebo dependencies:
 
 You can view lists of dependencies:
 
     conda search libsdformat --channel conda-forge --info
 
 Install dependencies, replacing `<#>` with the desired versions:
 
-    conda install libignition-math<#> libignition-tools<#> --channel conda-forge
+    conda install libgz-math<#> libgz-tools<#> --channel conda-forge
```
Comment (azeey) [inline install/tutorial_9_later.md:198]: There is currently no `libgz-math<#>` package on conda.

File: install/tutorial_12.md [hunk 5]
```diff
@@ -183,22 +183,22 @@ Miniconda suffices.
...
+Install Gazebo dependencies:
 
 You can view lists of dependencies:
 
     conda search libsdformat --channel conda-forge --info
 
 Install dependencies, replacing `<#>` with the desired versions:
 
-    conda install libignition-math<#> libignition-tools<#> --channel conda-forge
+    conda install libgz-math<#> libgz-tools<#> --channel conda-forge
```
Comment (azeey) [inline install/tutorial_12.md:201]: There is no `libgz-math<#>` package on conda.

**Review by azeey — CHANGES_REQUESTED** [review]

**Comment by ahcorde** [comment]
friendly ping @azeey @scpeters

File: install/tutorial_12.md (hunk 5 above)
Comment (traversaro) [reply install/tutorial_12.md]: There is now: https://github.com/conda-forge/gz-math-feedstock .

**Comment by ahcorde** [comment]
friendly ping @azeey @scpeters

File: install/tutorial_6-x.md [hunk 6]
```diff
@@ -33,7 +33,7 @@ for Ubuntu distros earlier than bionic, the OSRF package repository should be us
 
 for Ubuntu bionic and later, skip to here:
 
-    sudo apt-get install libignition-math4-dev
+    sudo apt-get install libgz-math4-dev
```
Comment (azeey) [inline install/tutorial_6-x.md:36]: Can you revert this? There is no `libgz-math4-dev`.

File: install/tutorial_9_later.md [hunk 7]
```diff
@@ -180,22 +180,22 @@ Miniconda suffices.
...
+Install Gazebo dependencies:
 
 You can view lists of dependencies:
 
     conda search libsdformat --channel conda-forge --info
 
 Install dependencies, replacing `<#>` with the desired versions:
 
-    conda install libignition-math<#> libignition-tools<#> --channel conda-forge
+    conda install libignition-math<#> libgz-tools<#> --channel conda-forge
```
Comment (azeey) [inline install/tutorial_9_later.md:198]: I think we should only use `libgz-*` packages for garden packages. Can you rever this and add a separate line for `libsdformat13` and later?

File: specify_pose/tutorial.md [hunk 8]
```diff
@@ -64,13 +64,13 @@ which both share the same definition of roll, pitch, and yaw angles:
 
 For a command-line utility to convert between roll-pitch-yaw angles,
 quaternions, and rotation matrices, please see the
-[quaternion\_from\_euler](https://github.com/ignitionrobotics/ign-math/blob/ign-math4/examples/quaternion_from_euler.cc)
-and [quaternion\_to\_euler](https://github.com/ignitionrobotics/ign-math/blob/ign-math4/examples/quaternion_to_euler.cc)
-example programs in ignition math.
+[quaternion\_from\_euler](https://github.com/gazebosim/gz-math/blob/ign-math4/examples/quaternion_from_euler.cc)
+and [quaternion\_to\_euler](https://github.com/gazebosim/gz-math/blob/ign-math4/examples/quaternion_yo_euler.cc)
```
Comment (azeey) [inline specify_pose/tutorial.md:68]:
```suggestion
and [quaternion\_to\_euler](https://github.com/gazebosim/gz-math/blob/ign-math4/examples/quaternion_to_euler.cc)
```

File: install/tutorial_12.md [hunk 9]
```diff
@@ -183,22 +183,22 @@ Miniconda suffices.
...
+Install Gazebo dependencies:
 
 You can view lists of dependencies:
 
     conda search libsdformat --channel conda-forge --info
 
 Install dependencies, replacing `<#>` with the desired versions:
 
-    conda install libignition-math<#> libignition-tools<#> --channel conda-forge
+    conda install libignition-math<#> libgz-tools<#> --channel conda-forge
```
Comment (azeey) [inline install/tutorial_12.md:201]: Can you revert this since for libsdformat12, we'd still need `libignition-*` packages.

**Review by azeey — APPROVED** [review]
